src/driver/iocp/mod.rs

```diff
@@ -112,14 +116,21 @@ pub trait OpCode {
     /// # Safety
     ///
     /// `self` must be alive until the operation completes.
-    unsafe fn operate(&mut self, optr: *mut OVERLAPPED) -> Poll<io::Result<usize>>;
+    unsafe fn operate(
```

Comment:
I don't agree with changing IOCP and mio driver to support reg fds. Let's make it io-uring specific.

Reply:
Do you mean that for IOCP/mio RegFd is a type alias for RawFd?

Reply:
I mean that IOCP/mio should never know about RegFd. It shouldn't have interface to accept/return RegFd.